crates/bevy_gltf/src/loader.rs

```diff
@@ -1102,7 +1107,7 @@ fn load_node(
     let mut morph_weights = None;
 
     node.with_children(|parent| {
-        if settings.load_meshes {
+        if !settings.load_meshes.is_empty() {
```

Comment:
This feels pretty unclear to me. Can we add a helper method on `RenderAssetUsages` to better capture the semantic meaning?